src/main/java/se/sundsvall/parkingpermit/businesslogic/worker/execution/HandleLostCardTaskWorker.java

```diff
@@ -0,0 +1,101 @@
+package se.sundsvall.parkingpermit.businesslogic.worker.execution;
+
+import static generated.se.sundsvall.casedata.NoteType.PUBLIC;
+import static generated.se.sundsvall.partyassets.Status.BLOCKED;
+import static java.util.Collections.emptyList;
+import static se.sundsvall.parkingpermit.Constants.CASEDATA_KEY_ARTEFACT_LOST_PERMIT_NUMBER;
+import static se.sundsvall.parkingpermit.Constants.CASE_TYPE_LOST_PARKING_PERMIT;
+import static se.sundsvall.parkingpermit.Constants.PARTY_ASSET_STATUS_ACTIVE;
+import static se.sundsvall.parkingpermit.Constants.PARTY_ASSET_TYPE;
+import static se.sundsvall.parkingpermit.Constants.ROLE_APPLICANT;
+
+import generated.se.sundsvall.casedata.Errand;
+import generated.se.sundsvall.casedata.ExtraParameter;
+import generated.se.sundsvall.casedata.Note;
+import generated.se.sundsvall.casedata.PatchErrand;
+import generated.se.sundsvall.casedata.Stakeholder;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import org.camunda.bpm.client.spring.annotation.ExternalTaskSubscription;
+import org.camunda.bpm.client.task.ExternalTask;
+import org.camunda.bpm.client.task.ExternalTaskService;
+import org.springframework.stereotype.Component;
+import se.sundsvall.parkingpermit.businesslogic.handler.FailureHandler;
+import se.sundsvall.parkingpermit.businesslogic.worker.AbstractTaskWorker;
+import se.sundsvall.parkingpermit.integration.camunda.CamundaClient;
+import se.sundsvall.parkingpermit.integration.casedata.CaseDataClient;
+import se.sundsvall.parkingpermit.service.PartyAssetsService;
+
+@Component
+@ExternalTaskSubscription("HandleLostCardTask")
+public class HandleLostCardTaskWorker extends AbstractTaskWorker {
+
+	private static final String NOTE_TEXT = "The asset with id %s has been blocked.";
```

Comment:
`"The asset with id %s has been blocked."`
->
`"The asset with ID '%s' has been blocked."`

?